### DISCUSSION TITLE
Error during archive xcodeproj

### DISCUSSION THREAD

**Discussion by jakfear863** [ionic-team/capacitor#5921] [Q&A]
Hi. 
First of all im working on IOS

Im trying to archive an xcodeproj manually but there is a error on "AppDelegate.swift" file:
"Import capacitor" - no such module 'Capacitor'

I have no idea at all about this type of things but i have to solve this for my work...
If you need more info, go ask me and ill try to answer as best as i can.

Can you help me? Thanks you and your time

**Comment by jakfear863** [comment]
I don't know if this type of post belongs here so im sorry if it is not

**Comment by jcesarmobile** [comment]
You have to open/build the `App.xcworkspace`, not the `App.xcodeproj`.

We have `npx cap open` command that opens the `App.xcworkspace` for you

**Reply by jakfear863** [reply]
> You have to open/build the `App.xcworkspace`, not the `App.xcodeproj`.
> 
> We have `npx cap open` command that opens the `App.xcworkspace` for you

thank you for answering and clarifying my question
I can't use that command because im using jenkins + xcode to automatice the process
I'm getting this error when it comes to the export of the IPA file but i think this is not because capacitor and these things.

```
Error Domain=IDEDistributionMethodManagerErrorDomain Code=2 "Unknown Distribution Error" UserInfo={NSLocalizedDescription=Unknown Distribution Error}
2022-09-16 11:18:06.795 xcodebuild[29012:1774934] [MT] IDEDistributionMethodManager: -[IDEDistributionMethodManager orderedDistributionMethodsForTask:archive:]: Error = Error Domain=IDEDistributionMethodManagerErrorDomain Code=2 "Unknown Distribution Error" UserInfo={NSLocalizedDescription=Unknown Distribution Error}
error: exportArchive: Found no compatible export methods for: <DVTFilePath:0x7fa0176eeeb0:'/Users/sdaae/datos_apl/jenkins-dsa/workspace/TEST_IOS/ios/App/App.xcarchive'>

Error Domain=IDEFoundationErrorDomain Code=1 "Found no compatible export methods for: <DVTFilePath:0x7fa0176eeeb0:'/Users/sdaae/datos_apl/jenkins-dsa/workspace/TEST_IOS/ios/App/App.xcarchive'>" UserInfo={NSLocalizedDescription=Found no compatible export methods for: <DVTFilePath:0x7fa0176eeeb0:'/Users/sdaae/datos_apl/jenkins-dsa/workspace/TEST_IOS/ios/App/App.xcarchive'>}
```

But again, thanks you for your time
